ethicalads-theme/templates/ea/advertisers-pricing.html

```diff
@@ -0,0 +1,242 @@
+{% extends "page.html" %}
+
+
+{% block content %}
+
+<section class="position-relative mt-5 mb-10">
+
+  <!-- Content -->
+  <div class="container">
+    <div class="row align-items-center text-center text-md-left">
+      <div class="col-12 col-md-6">
+
+        <!-- Image -->
+        <img src="/theme/img/advertiser-dashboard-in-laptop.png" alt="..." class="img-fluid mw-100 float-right mb-6 mb-md-0" data-aos="fade-right">
+
+      </div>
+      <div class="col-12 col-md-6">
+
+        <!-- Heading -->
+        <h1 class="display-3 font-weight-light" data-aos="fade-left">Fair and upfront pricing for your campaign</h1>
+
+        <!-- Text -->
+        <p class="font-size-lg" data-aos="fade-left" data-aos-delay="200">Our ad pricing is completely transparent, simple to understand, and not hidden behind a lead gate</p>
```

Comment:
```suggestion
        <p class="font-size-lg" data-aos="fade-left" data-aos-delay="200">Our ad pricing is completely transparent and simple to understand</p>
```

`lead gate` is weird in an H2, could also be 👍 with other copy there?